package.json

```
@@ -1433,6 +1433,11 @@
 				"command": "code-for-ibmi.cl.runSelected",
 				"title": "Run selected CL command",
 				"category": "IBM i"
+			},
+			{
+				"command": "code-for-ibmi.openTerminalHere",
+				"title": "Open PASE terminal here",
```

How about `Open terminal here` instead?

```suggestion
				"title": "Open terminal here",
```